Sources/ClientRuntime/PrimitiveTypeExtensions/URLQueryItem+Extensions.swift

```diff
@@ -3,8 +3,6 @@
  * SPDX-License-Identifier: Apache-2.0.
  */
 
-public typealias URLQueryItem = SDKURLQueryItem
-
```

Comment:
Since this typealias is removed, references to `ClientRuntime. URLQueryItem` everywhere must be changed to `SDKURLQueryItem`.